index.html

```diff
@@ -0,0 +1,104 @@
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:ital@0;1&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <div class="container-nav">
+          <a href="index.html" class="logo-title">Marlyn Mayienga</a>
+          <nav class="nav-bar">
+            <ul>
+              <li class="portfolio">
+                <a href="#portfolio">Portfolio </a>
+              </li>
+              <li>
+                <a href="#">About </a>
+              </li>
+              <li>
+                <a href="#contact">Contact </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <div class="hamburger-btn">
+            <i class="fas fa-bars"></i>
+          </div>
+      </header>
+      <section class="introduction">
+        <div class="container">
+        <div class="first-ellipse"></div>
+        <div class="ellipse_cut"></div>
+        <div class="second_img"></div>
+        <div class="first_img"></div>
+          <h1 class="greeting">I’m Merlin <br> Glad to see you!
+    </h1>
+          <p class="first-paragraph">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h2 class="connect">
+            LET’S CONNECT
+          </h2>
+          <div class="social-links">
+            <ul>
+              <li>
+                <a href="https://twitter.com/M_ayienga"
+                  ><i class="fab fa-twitter"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+                  ><i class="fab fa-linkedin"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.instagram.com/merl.__/?hl=en"
+                  ><i class="fab fa-medium-m"></i>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/M_ayienga"
+                  ><i class="fab fa-github"></i>
+                </a>
+              </li>
+              <li>
+                <a href="marlymayienga@gmail.com"
+                  ><i class="fab fa-angellist"></i
+                ></a>
+              </li>
+            </ul>
```

Comment:
While you did address the previous reviewer's requirement, you didn't do so properly. Your icons don't quite look like those from the template design. The color is quite darker and they're quite smaller, please fix them.